assets/default.game.yaml

```diff
@@ -12,6 +12,161 @@ main_menu:
     image: ui/main-menu-background.png
     image_size: [1919, 1027]
 
+default_input_maps:
```

Comment:
Very nice how clean it is to have analogs + dpad + keyboard all share input maps